.storybook/main.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  stories: ['../src/**/*.stories.(js|jsx|ts|tsx)'],
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+  ],
+  framework: '@storybook/react',
```

Comment:
why this line?

Reply:
@maneetgoyal sir this line denotes the type of framework that we are using, It helps `Storybook` to load and build processes specific to framework. 
Source: https://storybook.js.org/docs/react/configure/overview

Reply:
See if this should be added to d3 portfolio and app repo also, I think we aren't using it there but still things are working, try to research on this property helps and add it to the repos also if needed

Reply:
creating https://github.com/manufac-analytics/reactjs-calendar-heatmap/issues/8